apps/web-client/src/routes/debug/+page.svelte

```diff
@@ -89,11 +101,29 @@
 	$: dbName = $dbNamePersisted;
 
 	const populateDatabase = async function () {
-		const db = await getInitializedDB(dbName);
+		const { db } = await getInitializedDB(dbName);
 		errorMessage = null;
 		console.log("Populating database");
 		try {
-			await db.db.exec(debugData);
+			await upsertBooks(db, dd.books);
+			await upsertCustomers(db, dd.customers);
+			await upsertSuppliers(db, dd.suppliers);
+			await associatePublishers(db, dd.supplierPublishers);
+			// possible order lines
+			const isbns = dd.customerOrderLines.map((line) => line.isbn);
+			await addBooksToCustomer(db, dd.customers[0].id, isbns);
```

Comment:
This is adding all books to the first customer order regardless of the `customer_id` specified on the customer order line